tests/src/unit_v2_tests.cpp

```diff
@@ -0,0 +1,197 @@
+#include <cstdint>
+#include <ratio>
+#include <type_traits>
+
+#include <gtest/gtest.h>
+#include <stronk/stronk.h>
+#include <stronk/unit_v2.h>
+#include <stronk/utilities/dimensions.h>
+
+namespace twig::unit_v2::unit_tests
+{
+
+// Example code:
+
+// First we define our stronk types:
+struct Distance : unit<Distance>
```

Comment:
I have never really liked these examples, because "Distance", "Time", "Mass" and the others are not really units.

I guess when we have ratio it makes more sense to introduce types like `Meter(s)`, `Minute(s)` and so on, with nice conversions to `KilometerPerHour` or similar.